Caesar_cipher/gonvers/main.py

```diff
@@ -0,0 +1,45 @@
+class CaesarCypher():
+    chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    chars_len = len(chars)
+    def __init__(self, key=0):
+        self.key = key
+    def set_key(self, key):
+        self.key = key
+    def get_key(self):
+        return self.key
+    def encrypt(self, string):
+        output = ""
+        for char in string.upper():
+            if char in self.chars:
+                output += self.chars[(self.key+self.chars.index(char))%self.chars_len]
+            else:
+                output += char
+        return output
+    def decrypt(self, string):
+        self.key *= -1
+        output = self.encrypt(string)
+        self.key *= -1
+        return output
+    
+
+if __name__ == "__main__":
+    cypher = CaesarCypher()
+    while(True):
+        match input("Do you want to : (e)ncrypt, (d)ecrypt, change the (k)ey or (q)uit\n").lower():
```

Comment:
this line is too long. standard according to pep8 is 79 characters
https://www.flake8rules.com/rules/E501.html